packages/playground/wordpress/src/rewrite-rules.ts

```diff
@@ -1,11 +1,164 @@
+ * RewriteCond %{REQUEST_FILENAME} -d
+ * RewriteRule ^ - [L]
+ * RewriteRule ^([_0-9a-zA-Z-]+/)?(wp-(content|admin|includes).*) $2 [L]
+ * RewriteRule ^([_0-9a-zA-Z-]+/)?(.*\.php)$ $2 [L]
+ * RewriteRule . index.php [L]
+ * ```
+ *
+ * # Multisite living at /scope:ambitious-chic-country/
+ *
+ * ```apache
+ * RewriteBase /scope:ambitious-chic-country/
+ * RewriteRule ^index\.php$ - [L]
+ *
+ * // Add a trailing slash to /wp-admin
+ * RewriteRule ^([_0-9a-zA-Z-]+/)?wp-admin$ $1wp-admin/ [R=301,L]
+ *
+ * RewriteCond %{REQUEST_FILENAME} -f [OR]
+ * RewriteCond %{REQUEST_FILENAME} -d
+ * RewriteRule ^ - [L]
+ *
+ * // The `wordpress/` prefix matches the document root, but seeing
+ * // it here is unexpected. @TODO: Why is it being added by WordPress?
+ * RewriteRule ^([_0-9a-zA-Z-]+/)?(wp-(content|admin|includes).*) wordpress/$2 [L]
+ * RewriteRule ^([_0-9a-zA-Z-]+/)?(.*\.php)$ wordpress/$2 [L]
+ * RewriteRule . index.php [L]
+ * ```
+ *
+ * ## .htaccess syntax
+ *
+ * Here's an excerpt/summary from the .htaccess documentation [^1][^2] for
+ * convenience:
+ *
+ *     The mod_rewrite module uses a rule-based rewriting engine, based
+ *     on a PCRE regular-expression parser, to rewrite requested URLs on
+ *     the fly. By default, mod_rewrite maps a URL to a filesystem path.
+ *     However, it can also be used to redirect one URL to another URL,
+ *     or to invoke an internal proxy fetch.
+ *
+ *     ## RewriteBase Directive
+ *
+ *     The RewriteBase directive specifies the URL prefix to be used for
+ *     per-directory (htaccess) RewriteRule directives that substitute a
+ *     relative path.
+ *
+ *     Syntax:
+ *          RewriteBase URL-path
+ *
+ *     (Setting RewriteBase to "/" makes it possible to use RewriteRule
+ *      patterns that **do not** start with a slash.)
+ *
+ *     ## RewriteRule Directive
+ *
+ *     Defines rules for the rewriting engine.
+ *
+ *     Syntax:
+ *          RewriteRule Pattern Substitution [flags]
+ *
+ *     ## Flags
+ *
+ *        - L|Last
+ *            Stop processing the rule set. In most contexts, this means
+ *            that if the rule matches, no further rules will be processed
+ *
+ *        - NC|No Case
+ *            Ignore case when matching.
+ *
+ *        - R|Redirect
+ *            Causes a HTTP redirect to be issued to the browser.
+ *
+ * ## Differences with .htaccess
+ *
+ * [1] https://httpd.apache.org/docs/current/rewrite/intro.html
+ * [2] https://httpd.apache.org/docs/current/rewrite/flags.html
  */
 export const wordPressRewriteRules: RewriteRule[] = [
+	/**
+	 * Substitutes the multisite WordPress rewrite rule:
+	 *
+	 * RewriteBase /
+	 * RewriteRule ^([_0-9a-zA-Z-]+/)?(wp-(content|admin|includes).*) $2 [L]
+	 */
 	{
-		match: /^\/(.*?)(\/wp-(content|admin|includes)\/.*)/g,
+		match: new RegExp(
+			/* The .htaccess rule does not have an explicit initial slash,
+				but it's still implied by `RewriteBase /` */
+			`^(/[_0-9a-zA-Z-]+)?` +
```

Comment:
Why did this change from
`^\/(.*?)`
to 
`^(/[_0-9a-zA-Z-]+)?`

Are there paths that worked before that will not work now?